readme.md

```diff
@@ -24,6 +24,7 @@ src/gz hackypackages http://<SERVER_URL>:<PORT>
 1. run `opkg update` to update local package list
 1. run `opkg install <PACKAGE_NAME>` to install a package
 
+
```

Comment:
ein Abschnitt "How to contribute" mit einem Link auf das entsprechende markdown wäre praktisch